mne/epochs.py

```diff
@@ -850,10 +850,12 @@ def ch_names(self):
 
     @copy_function_doc_to_method_doc(plot_epochs)
     def plot(self, picks=None, scalings=None, n_epochs=20, n_channels=20,
-             title=None, show=True, block=False):
+             title=None, events=None, event_colors=None, show=True,
```

Comment:
I believe the convention is (or should be) to have show, block and verbose as the last kwargs.